test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,17 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, new Integer(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+        // some null objects
+        assertHasNull(null, null, null);
+        assertHasNull("abc", 1, 2, 3, null);
+        assertHasNull(null, 4, "bnm", 3, null);
+        assertHasNull("zxc", "qwe", 2, 3, 4, null);
```

Comment:
Quite thorough, however many redundancy.
Does each additional test case have chance of finding a new bug?